flixel/input/mouse/FlxMouseEventManager.hx

```diff
@@ -443,22 +443,30 @@ class FlxMouseEventManager extends FlxBasic
 		for (camera in Register.object.cameras)
 		{
 			#if FLX_MOUSE
-			_point = FlxG.mouse.getWorldPosition(camera, _point);
-			
-			if (checkOverlapWithPoint(Register, _point, camera))
+			_point = FlxG.mouse.getScreenPosition(camera, _point);
```

Comment:
i've found that if we change this _point caclulation code `_point = FlxG.mouse.getScreenPosition(camera, _point);` to something like this:
``` 
public function getPositionInCameraView(?Camera:FlxCamera, ?point:FlxPoint):FlxPoint
	{
		if (Camera == null)
			Camera = FlxG.camera;
		
		if (point == null)
			point = FlxPoint.get();
		
		point.x = (_globalScreenX - Camera.x) / Camera.zoom;
		point.y = (_globalScreenY - Camera.y) / Camera.zoom;
		
		return point;
	}
```
then checks working properly.

This new method calculates position of a point relative to camera view rectangle